conans/client/build/msbuild.py

```diff
@@ -241,7 +240,7 @@ def format_macro(name, value):
     @staticmethod
     def get_version(settings):
         msbuild_cmd = "msbuild -version"
-        vcvars = tools_vcvars_command(settings)
+        vcvars = vcvars_command(settings)
```

Comment:
This function requires an `output` or it will use the `global_output` and will raise/print a warning (CI won't fail because we have removed it with the CONAN_V2_MODE PR, that PRs would introduce too many catch-warnings otherwise).

This is the underlying reason to use the `tool` here, to avoid that warning, and it is still valid. We can move the import inside this function and add a comment.

Reply:
Got it, fixed it.
We certainly have to improve the tools injection for Conan 2.0.